### ISSUE TITLE
add p-value, chr, and base pair location to DB?

### ISSUE THREAD

**Issue by vjcitn** [ccb-hms/GWASCatalogSearchDB#5]
Hi @rsgoncalves -- I think we can get rid of the gwascat snapshot if you add the location and stat significance measure into the main table.  It looks like you have one record per GCST study accession, but the catalog records all associations.  We could use a thin table with just chromosome, base-pair, p-value, and rsnumber and I think we would be good to go.  Note that I just reviewed the p-values, found one that EBI is publishing as 6e+12 (i.e., forgot the minus sign).  I have notified them.

**Comment by rsgoncalves** [comment]
Hi Vince. Indeed in the gwascatalog_metadata table we have one record per GCST study accession— this is EBI's "studies" table. I can include the "associations" table also — perhaps a trimmed-down version of it, with the columns for the details you listed. Can you confirm that this is the right table to look at: `All associations v1.0.2 - with added ontology annotations, GWAS Catalog study accession numbers and genotyping technology` (https://www.ebi.ac.uk/gwas/docs/file-downloads).

**Comment by rsgoncalves** [comment]
This is tentatively closed via [v0.7.0](https://github.com/ccb-hms/GWASCatalogSearchDB/releases/tag/v0.7.0) release. 

@vjcitn please see if you can have a look at the newly added associations table in the DB and let me know how things look.

**Comment by vjcitn** [comment]
Hi -- this is very nice, all the locations and p-values are now available and are easily extracted.  But because the phenotypes are collapsed to the study level, we lose information associating SNP with phenotype. 

```
DataFrame with 7 rows and 4 columns
  STUDY ACCESSION      CHR_ID   CHR_POS           MAPPED_TRAIT
      <character> <character> <numeric>            <character>
1      GCST002955           6 137629910 forced expiratory vo..
2      GCST002955          13  66152771 response to biologic..
3      GCST002955           8  40475802 forced expiratory vo..
4      GCST002955           2 153874774 forced expiratory vo..
5      GCST002955           5  58912391 response to gases an..
6      GCST002955           7 150813827 response to gases an..
7      GCST002955           5 165680095 response to gases an..
```
For that study the database has
```
   STUDY.ACCESSION
9       GCST002955
10      GCST002955
11      GCST002955
                                                                             DISEASE.TRAIT
9  Forced expiratory volume in 1 second (occupational environmental exposures interaction)
10 Forced expiratory volume in 1 second (occupational environmental exposures interaction)
11 Forced expiratory volume in 1 second (occupational environmental exposures interaction)
                                                                                                                              MAPPED_TRAIT
9  response to biological dust exposure, response to gases and fumes exposure, forced expiratory volume, response to mineral dust exposure
10 response to biological dust exposure, response to gases and fumes exposure, forced expiratory volume, response to mineral dust exposure
11 response to biological dust exposure, response to gases and fumes exposure, forced expiratory volume, response to mineral dust exposure
                       MAPPED_TRAIT_URI MAPPED_TRAIT_CURIE
9  http://www.ebi.ac.uk/efo/EFO_0006994        EFO:0006994
10 http://www.ebi.ac.uk/efo/EFO_0004314        EFO:0004314
11 http://www.ebi.ac.uk/efo/EFO_0006993        EFO:0006993
```
but AFAICT the connection of trait to SNP is lost.  If you kept the MAPPED_TRAIT_CURIE with the p-values I think we would be all set.

**Comment by rsgoncalves** [comment]
Thank you for pointing this out Vince. Indeed the connection between traits and SNPs was not preserved in the associations table. 

I modified the DB now to contain the CURIE(s) associated with each SNP in the associations table. When there are multiple (comma-separated) traits in MAPPED_TRAIT, we currently generate also multiple comma-separated CURIEs in the same row. I suspect this choice might hinder queryability, what do you think? Alternatively we could pull out the essential columns from the associations table to have a slimmer table with a single EFO term for every row.

**Comment by rsgoncalves** [comment]
closed since release [v0.7.1+](https://github.com/ccb-hms/GWASCatalogSearchDB/releases/tag/v0.7.1)
